src/holofuel/pages/CreateOffer/CreateOffer.js

```diff
@@ -131,17 +144,18 @@ export default function CreateOffer ({ history: { push } }) {
         name='notes'
         placeholder='Notes'
         ref={register} />
+
       <RecentCounterparties
         styleName='recent-counterparties'
         agents={agents}
         selectedAgentId={counterpartyId}
         selectAgent={selectAgent} />
-      <Button type='submit' wide variant='secondary' styleName='send-button'>Send</Button>
+      <Button type='submit' wide variant='secondary' styleName='send-button' disabled={counterpartyId.length === AGENT_ID_LENGTH ? disabled : true}>Send</Button>
```

Comment:
`disabled = {counterpartyNotFound || counterpartyId.length !== AGENT_ID_LENGTH}`

Reply:
So I need to leave the ternary operator for the case wherein the  counterpartyId length is less than full amount and the counterpartyNotFound hasn't been run yet (as it is only trigged once the counterpartyId length is satisfied)